typing/jkind.ml

```diff
@@ -3098,11 +3098,10 @@ let is_void_defaulting = function
    any, and modal upper bounds are constant. *)
 (* The choice of [type_equal] and [jkind_of_type] doesn't matter because there
    are no with-kinds on the left-hand kind. *)
-let is_max jkind =
-  sub
-    ~type_equal:(fun _ _ -> false)
-    ~jkind_of_type:(fun _ -> None)
-    Builtin.any_dummy_jkind jkind
+let is_obviously_max = function
```

Comment:
plausibly we should add some more documentation here - or in the signature - explaining what "obviously" means